content/blog/2020-07-31-spreadsheet-workflows-using-r/index.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: Spreadsheets Workflows in R
+date: '2020-07-31'
+slug: spreadsheets-using-r
+categories:
+  - learn
+tags:
+  - spreadsheets
+  - tidyverse
+author:
+  - Devin Johnson
+photo:
+  url: https://unsplash.com/photos/Wpnoqo2plFA
+  author: Mika Baumeister
+---
+
+There's so much to discover and learn when working with spreadsheets, and creating a resource that speaks to both spreadsheet and R users has been an exciting challenge.  While my internship has focused on a plethora of topics from comparing numerous R package functionalities that work with spreafsheets, to writing tips and tricks on best practices for working in both spreadsheets and RStudio, what I'd like to focus on in this post is a section dedicated to the R versions of common spreadsheet workflows.
+
+To do that we'll assume the role of a data analyst in two hypothetical scenarios and walkthrough the tidyverse equivalent of some common analyes and tasks one is likely to come across when working in spreadsheets.   
```

Comment:
```suggestion
To do that we'll assume the role of a data analyst in two hypothetical scenarios and walk through the tidyverse equivalent of some common analyses and tasks one is likely to come across when working in spreadsheets.   
```

I think a spell check pass would be good (better than making individual edits like this).

Reply:
Noted. I think I was working on a slightly older version of RStudio that didn't have the spellcheck in markdown, updated now so we should be good on that front.